hack/operatorhub/main.go

```diff
@@ -595,11 +600,17 @@ func getImageDigest(apiKey, projectId, version string) (string, error) {
 	req.Header.Set("X-API-KEY", apiKey)
 
 	q := req.URL.Query()
-	q.Add("filter", fmt.Sprintf("repositories.tags.name==%s;deleted==false", version))
+	// This filter does not work any longer.  Redhat case is open to find working filter.
```

Comment:
Maybe track this problem with an issue, and add reference it here.
```suggestion
	// "repositories.tags.name" filter does not work any longer.  See https://github.com/elastic/cloud-on-k8s/issues/XXXX.
```